reStream.sh

```diff
@@ -86,6 +79,65 @@ if ! ssh_cmd true; then
     exit 1
 fi
 
+rm_version="$(ssh_cmd cat /sys/devices/soc0/machine)"
+
+# We cannot read the framebuffer directory in the reMarkable 2, but we know the
+# address of the pointer to the framebuffer for some reMarkable 2 versions.
+# This will need to be changed each time xochitl updates.
+rm2_getpointer() {
+    xochitl_version="$(ssh_cmd "sed -n 's/^REMARKABLE_RELEASE_VERSION=//p' /usr/share/remarkable/update.conf")"
```

Comment:
/sys/devices/soc0/machine might be a better indicator, possibly

Reply:
Based on https://github.com/Eeems/oxide/issues/48#issuecomment-716523459 I would agree

Reply:
This is is not to find out if we're running rM1 or 2, but to find the correct pointer between `xochitl` versions. That information is not included in `/sys/devices/soc0/machine`?